Add 4.29.0.Final to tools.jboss.org (#1002)
jbosstools/jbosstools-website: blog/4.29.0.final.adoc

= JBoss Tools 4.29.0.Final for Eclipse 2023-09
:page-layout: blog
:page-author: sbouchet
:page-tags: [release, jbosstools, jbosscentral]
:page-date: 2023-11-02

Happy to announce 4.29.0.Final build for Eclipse 2023-09.

Downloads available at link:/downloads/jbosstools/2023-09/4.29.0.Final.html[JBoss Tools 4.29.0 Final].

== What is New?

Full info is at link:/documentation/whatsnew/jbosstools/4.29.0.Final.html[this page]. Some highlights are below.

=== General

==== Components Removal

> As planned and communicated in a previous blog article, the following components have been

Reviewer: ```suggestion
As outlined and announced in a prior blog post, the subsequent components have been excluded from the latest JBoss Tools distribution:```

removed from the JBoss Tools distribution:

- WebServices
- Batch

=== Hibernate Tools

== Hibernate Runtime Provider Updates

> A number of additions and updates have been performed on the available Hibernate runtime  providers.

Reviewer: ```suggestion
Multiple additions and updates have been performed on the available Hibernate runtime providers.
```

=== New Runtime Provider

A new Hibernate 6.3 runtime provider incorporates Hibernate Core version 6.3.1.Final, Hibernate Ant version 6.3.1.Final and Hibernate Tools version 6.3.1.Final.

=== Runtime Provider Updates

The Hibernate 6.2 runtime provider now incorporates Hibernate Core version 6.2.13.Final, Hibernate Ant version 6.2.13.Final and Hibernate Tools version 6.2.13.Final.

The Hibernate 5.3 runtime provider now incorporates Hibernate Core version 5.3.32.Final and Hibernate Tools version 5.3.32.Final.

=== Server Tools

==== New Server view based on RSP

> A new view using RSP (Remote Server Protocol) as been added in JBossTools, called "RSP Servers View".

Reviewer: ```suggestion
In JBoss Tools, a fresh perspective has been introduced, known as the "RSP Servers View," which utilizes the Remote Server Protocol (RSP).```

image::/documentation/whatsnew/server/images/RSP_View_1.png[Empty RSP View at start]

> First, the server connector used to handle the servers needs to be started. Right click on any Server connector entry, and select "Start RSP".

Reviewer: ```suggestion
To begin, you must initiate the server connector responsible for managing the servers. Simply right-click on any entry for the Server connector, and choose the "Start RSP" option.
```

The server connector is now starting and it's status will change to [STARTED].

image::/documentation/whatsnew/server/images/RSP_View_2.png[Empty RSP View started]

You can then choose between download a server or use a local server.

> Let's create a new server. Right click on the started Server connector and select "Download Server". Select any runtime on the list and click OK.

Reviewer: ```suggestion
Now, let's create a new server. Simply right-click on the active Server connector and select "Download Server." Choose any runtime from the list and confirm by clicking OK.
```
